packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -245,6 +250,9 @@ const styles: ComponentStyles<SiteManagedClasses, DevSiteDesignSystem> = {
         alignSelf: "flex-start",
     },
     site_paneToggleButtonIcon: {
+        "& svg": {
```

Comment:
Is there a reason we are targeting all SVG elements?

Reply:
Removed and added fill directly on `site_paneToggleButtonIcon`.